hw9/App.java

```diff
@@ -0,0 +1,23 @@
+package org.hillel.hw9;
+
+import org.hillel.hw9.exceptions.ArrayDataException;
+import org.hillel.hw9.exceptions.ArraySizeException;
+
+import static org.hillel.hw9.ArrayValueCalculator.doCalc;
+
+public class App {
+    // In commented code I show another method of throwing exceptions

```

Comment:
Instead of leaving comments here, would be better to implement another version of this task, put it in a separate package, and give me a hint in Telegram or LMS, ok?)